pydocstringformatter/utils/argument_parsing.py

```diff
@@ -19,24 +20,40 @@ def _parse_command_line_arguments(
 def _register_arguments(version: str) -> argparse.ArgumentParser:
     """Create an argument parser and add all supported arguments"""
     parser = argparse.ArgumentParser(prog="pydocstringformatter")
-
     parser.add_argument("files", nargs="*", type=str)
-
     parser.add_argument(
         "-w",
         "--write",
         action="store_true",
         help="Write the changes to file instead of printing the files to stdout",
     )
-
     parser.add_argument(
         "-v",
         "--version",
         action="version",
         version=version,
         help="Show version number and exit",
     )
+    return parser
+
 
+def _register_arguments_formatters(
+    parser: argparse.ArgumentParser, formatters: List[Formatter]
+) -> argparse.ArgumentParser:
+    """Register a list of formatter, so they can all be deactivated or activated."""
+    for formatter in formatters:
+        help_text = f"ctivate the {formatter.name(delimiter=' ')} formatter"
+        name = formatter.name()
+        parser.add_argument(
+            f"--{name}",
+            action="store_true",
+            dest=name,
+            default=not formatter.optional,
+            help=f"A{help_text} : {formatter.__doc__}",
```

Comment:
I like this, but this should probably be documented somewhere in the "Contributing guidelines". This is crucial information to know when writing a new formatter.

Reply:
Done in latest commit